<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Modificar_cliente</class>
 <widget class="QDialog" name="Modificar_cliente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>431</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Nombre</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="Nombre_MC">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Apellido</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="Apellido_MC">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Edad</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="Edad_MC">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="Validar_MC">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Modificar Cliente</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Por favor ingrese los datos a modificar:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Identificacion</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Id_MC">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>30</y>
     <width>291</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="validar">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Validar</string>
   </property>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>label</zorder>
  <zorder>label</zorder>
  <zorder>label_5</zorder>
  <zorder>Id_MC</zorder>
  <zorder>validar</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
